### ISSUE TITLE
When broadcasting with iphone in safari the quality of the video is really bad mainly when moving it gets pixelated.

### ISSUE THREAD

**Issue by CastriOnlive** [millicast/millicast-sdk#178]
**Version affected**
latest v0.1.31

**Describe the bug**
When broadcasting with iphone in safari the quality of the video is really bad compared to some months ago.

**To Reproduce**
1- start publishing with an iphone
2- watch the published stream in the hosted viewer or from the sdk viewer.

**Expected behavior**
A good video quality without pixelation as in android.

**Screenshots**
![image](https://user-images.githubusercontent.com/85135004/181791312-5981a470-c4db-413c-95f3-9eafd1756a13.png)

**Environment (please complete the following information):**
 - OS: iOS
 - Browser safari
 - Version:  iphone 11, safari 14
 
**Additional context**
No more context

**Comment by fcancela** [comment]
Hey, we haven't been able to reproduce your bug so far. 

We tried tested it with different devices (iPhone 13 and SE), codecs (h264, vp8), and FPS (30, 60) without reproducing the behaviour you are having. 

Have you tested it with another iPhone?
Is this happening consistently?
How mediaStats look like? 
Do you have simulcast activated?

Maybe this information can help narrow down your problem.

Thanks in advance!

**Comment by CastriOnlive** [comment]
yes in iphone 13 we do not have the problem, it seems to be on iphone 11. can you please try with an iphone 11?

**Comment by fcancela** [comment]
We don't have specifically an iPhone 11. But we we were able to reproduce it with an iPhone SE. 

It may be probable that the bandwidth for the video being published is getting limited with Default configuration (we need to run some more tests in order to confirm this). 

We will keep you posted and will try to look for a WA for this issue.
